ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsSet.scala

```diff
@@ -27,7 +30,7 @@ object MetricsSet {
       recordTimeFunction = _.transactions.collect {
         case t if t.effectiveAt.isDefined => t.getEffectiveAt
       },
-      objectives = objectives,
+      configO = objectives,
```

Comment:
it was closer to the truth with `objectives` no?

Reply:
It was confusing because it was as if we're creating objectives from objectives while more truthful is that we're creating objectives from their config.
I could name it `objectivesConfigO` but that's already given by the config type `configO: Option[TransactionObjectives]`

Reply:
no biggie, but it just looked strange to see it changed from `objectives = objectives` to `configO = objectives`. I would have expected both to be adapted or none

Reply:
Changing in both places